lib/rspec/sidekiq/matchers/be_retryable.rb

```diff
@@ -11,7 +11,8 @@ def initialize(expected_retry)
         end
 
         def description
-          if @expected_retry.is_a?(Fixnum)
+          num_class = 1.class # Ruby 2.4 returns Integer, older returns Fixnum
+          if @expected_retry.is_a?(num_class)
```

Comment:
Yeah I am fine with however you guys want to change this. I'm not religious when it comes to style on the patch.